_ml-commons-plugin/prebuilt-models.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Pre-trained models
+parent: Model-serving framework
+nav_order: 120
+---
+
+The model-serving framework supports a variety of open-source pre-built models that can assist with a variety of machine learning (ML) search and analytics use-cases. 
+
+## Uploading pre-trained models
+
+To use a pre-trained model in your OpenSearch cluster:
+
+1. Select the model and download the model artifact and config.json file. For a list of pre-trained models, see [supported pre-trained model](#supported-pre-trained-models).
+2. Upload the model using the [upload API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework#upload-model-to-opensearch). For more information on how to upload and use model, see [Model-serving framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework).
+
+## Supported pre-trained models
+
+The model-serving framework supports the following models. 
+
+Although models might have similar use cases, they many of them contain different performance metrics and model sizes. For a comparison of the performances of pre-trained models, see the [sbert documentation](https://www.sbert.net/docs/pretrained_models.html#model-overview).
+{: .note}
+
+
+### Sentence transformers
+
+Sentence transformer models map sentences and paragraphs across a dimensional dense vector space. The number of vectors depends on the model. Use these models for use cases such as clustering and semantic search.
+
+| **Model name** | **Vector dimensions** | **Torchscript artifact** | **ONNX artifact** |
+|---|---|---|---|
+| `sentence-transformers/all-distilroberta-v1` | 768 dimensional dense vector space. | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-distilroberta-v1/1.0.1/torch_script/sentence-transformers_all-distilroberta-v1-1.0.1-torch_script.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-distilroberta-v1/1.0.1/torch_script/config.json) | - [model_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-distilroberta-v1/1.0.1/onnx/sentence-transformers_all-distilroberta-v1-1.0.1-onnx.zip)<br>- [config_url](https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/all-distilroberta-v1/1.0.1/onnx/config.json) |
```

Comment:
Is it correct to use periods for the data in column 2? E.g.:

```
768 dimensional dense vector space.
```

That isn't a sentence - should we still use periods?

Reply:
I'll defer to Nate Bower on this one.

Reply:
Added. I linked to the model-serving framework forum.